srcpkgs/fzf/template

```diff
@@ -22,6 +22,7 @@ post_install() {
 
 	sed -i -e 's#source ~/\.fzf\.bash; ##' shell/key-bindings.bash
 	vinstall plugin/fzf.vim 644 usr/share/vim/vimfiles/plugin
+	vinstall plugin/fzf.vim 644 usr/share/nvim/runtime/plugin
```

Comment:
Could this be a symlink to the `fzf.vim` file in the vim folder?

Reply:
It could, but then how `noextract=` would work in this case?

Reply:
Honestly I don't expect anyone to `noextract` it :P 

But I'm indeed not sure what the best way of doing it, feel free to do it how you prefer.i

Reply:
Sincerely, I prefer to just install it alongside the Vim one, it's less prone to error, I think.